ethcore/src/evm/evm.rs

```diff
@@ -70,6 +73,12 @@ impl From<Box<trie::TrieError>> for Error {
 	}
 }
 
+impl From<builtin::Error> for Error {
+	fn from(err: builtin::Error) -> Self {
+		Error::BuiltIn(err.0)
```

Comment:
Why not just have builtins return a `String` as error rather than go through the trouble of making a `builtin::Error` type which is just destructured here? Not sure it's worth allocating the string at all, actually, since
  - it's easy to make builtins fail
  - builtins are on a hot path
  - errors are silently swallowed anyway